abstract.md

```diff
@@ -8,9 +8,9 @@ used in game-playing programs). We conclude that since modularity is the
 key to successful programming, functional programming offers important
 advantages for software development.
 
-Поскольку програмное обеспечение становится все более сложным, все большую важность приобретает хорошая структура приложения.
-Хорошо структурированные программы легки в написании и отладке. В их основе лежит коллекция модулей, которые могут быть повторно использованы, чтобы снизить будущие затраты на программирование.
-В этой статье мы покажем, что две возможности функциональных языков, функции высшего порядка и ленивые вычисления, могут в значительной мере способствовать модульности.
-В качестве примеров, мы будем манипулировать списками и деревьями, запрограммируем несколько численных алгоритмов и реализуем альфа-бета эвристику (алгоритм из области искусственного интеллекта, используется в компьютерных играх).
-Мы пришли к выводу, что, поскольку модульность является ключом к успешному написанию программ, функциональное программирование предлагает важные преимущества для разработки програмного обеспечения.
+Поскольку програмное обеспечение становится все более и более сложным, все большую важность приобретает хорошая структура приложения.
+Хорошо структурированные программы легки в написании и отладке и позволяют на их основе создать коллекцию модулей, которые могут быть повторно использованы, чтобы снизить будущие затраты на разработку.
+В этой статье мы покажем, что две возможности функциональных языков, в частности, функции высшего порядка и ленивые вычисления, могут в значительной мере способствовать созданию модульности.
+В качестве примеров мы будем производить манипуляции со списками и деревьями, запрограммируем несколько численных алгоритмов и реализуем альфа-бета эвристику (алгоритм из области искусственного интеллекта, использующийся в компьютерных играх).
+Мы придем к выводу, что, поскольку модульность является ключом к успешному написанию программ, функциональное программирование предлагает важные преимущества для разработки програмного обеспечения.
```

Comment:
Вариант @goplon читается лучше, но кажется, искажает причинно следственную связь из текста. У тебя поулчилось 2 независимых утверждения, хотя в тексте явно указано, что "since modularity is the key to successful programming, functional programming offers important advantages"